src/components/layout/Footer.js

```diff
@@ -144,7 +152,9 @@ export default function Footer() {
     <FooterContainer>
       <ContentContainer>
         <LeftContent>
-          <Logo />
+          <LogoContainer>
+            <LogoText />
+          </LogoContainer>
```

Comment:
Can we just drop LogoContainer and wrap LogoText with the margin-bottom and width instead?

Reply:
reverted to previous version without container